polaris-react/src/components/LegacyCard/LegacyCard.scss

```diff
@@ -59,6 +59,11 @@
         border-bottom-left-radius: var(--p-border-radius-3);
         border-bottom-right-radius: var(--p-border-radius-3);
       }
+
+      // stylelint-disable-next-line -- se23 override
+      .Section p:first-of-type {
+        max-width: 80ch;
+      }
```

Comment:
I don't think this selector will work well as a lot of the subtitles in the settings could be outside of a `.Section`. This would also apply to subsequent sections that aren't directly under actions.

Maybe `p:first-child`? I still have to test this tho and think of edge cases